### PR TITLE
Use default sidebar for all tabs

### PR DESCRIPTION
**Pull request by JPry**
- **:fast_forward: https://github.com/the-events-calendar/tribe-common/commit/0e62dded808175ee53ff44ef130e32902ddf48ab**
- **Delete general sidebar, create default sidebar**
- **Register default sidebar**

### 🎫 Ticket

[TICKET_ID]

### ✔️ Checklist
- [ ] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: src/Tribe/Admin/Settings.php [hunk 1]
```diff
@@ -5,14 +5,12 @@
  * Manages the admin settings UI in relation to events configuration.
  */
 
+use Tribe\Admin\Troubleshooting as Troubleshooting;
```
[the remark this answers is not in the document]
Comment (Camwyn) [reply src/Tribe/Admin/Settings.php:8]:
```suggestion
use Tribe\Admin\Troubleshooting;
```

File: src/Tribe/Admin/Settings.php [hunk 2]
```diff
@@ -5,14 +5,12 @@
...
 use Tribe__App_Shop;
-use Tribe__Settings;
+use Tribe__Events__Main as Plugin;
```
Comment (Camwyn) [inline src/Tribe/Admin/Settings.php:10]: You shouldn't need this for the `tribe()` call later

**Review by Camwyn — CHANGES_REQUESTED** [review]
Small things

File: src/Tribe/Admin/Settings.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (JPry) [reply src/Tribe/Admin/Settings.php:8]: @Camwyn That's what I get for moving things around 😆

File: src/Tribe/Admin/Settings.php (hunk 2 above)
Comment (JPry) [reply src/Tribe/Admin/Settings.php:10]: I just sorted the `use` statements.

**Review by Camwyn — APPROVED** [review]

**Pull request merged**